server/src/main/scala/controllers/Application.scala

```diff
@@ -247,27 +277,51 @@ class Application @Inject()(
   ctrl =>
   val log = system.log
 
-  val chromaFetcher = new ProdChroma {
-    def system = ctrl.system
-  }
 
-  val apiService = new ApiService {
-    implicit val timeout = Timeout(5 seconds)
+  val chromafetcher = new ChromaFetcher with ProdSendAndReceive {
+    implicit val system: ActorSystem = ctrl.system
+  }
+  val portCode = sys.env.get("PORT_CODE")
+  log.info(s"PORT_CODE::: ${portCode}")
```

Comment:
what does this tell me? we can take it from the typesafe config, for consistency.